### ISSUE TITLE
Search in the IDE

### ISSUE THREAD

**Issue by sampsonij** [tingbot/tide-electron#50]
Really needs a search feature so you can find the code you are working on

**Comment by joerick** [comment]
Agreed. The Ace code editor provides one, so it shouldn't be a lot of work to enable/configure it, though I've not looked in detail. PRs welcome!
